plugins/config/torrenttv.py

```diff
@@ -9,10 +11,552 @@
 tvgurl = 'http://api.torrent-tv.ru/ttv.xmltv.xml.gz'
 
 # Shift the TV Guide time to the specified number of hours
-tvgshift = 0
+tvgshift = 24
```

Comment:
Yesterday the tvg from torrent-tv had a time lag in 24 hours. I've reported them this issue and seems it's fixed now.

Let me rebase this commit and change the default value.